cmd/pool/order.go

```diff
@@ -100,6 +100,11 @@ var baseBidFlags = []cli.Flag{
 			"from our account into the channel; can be " +
 			"used to create up to 50/50 balanced channels",
 	},
+	cli.BoolFlag{
+		Name: "private_channel",
+		Usage: "signal if the new channel should be announced to " +
+			"the network or not.",
```

Comment:
micro-nit: it looks like we don't use punctuation for command line flag descriptions.